src/index.js

```diff
@@ -115,7 +120,10 @@ function getChange(params) {
     return new Change(opts);
   }
   if (params.path) {
-    return new Change({ path: addLeadingSlash(params.path) });
+    return new Change({
+      path: addLeadingSlash(params.path),
+      time: new Date().toISOString(),
```

Comment:
When indexing a document manually (not through observation), we take the current time as fallback.